ordered_product/models.py

```diff
@@ -44,6 +44,7 @@ def decrease_quantity(self, qn):
 
     def increase_quantity(self, qn):
         self.quantity += qn
+        self.save()
```

Comment:
Is this fixing a bug? If so:
1. A bug usually means we were missing a test case earlier. So, you need to recreate this bug with a new test case (e.g. verify the quantity is updated in the db rather than in the object member variable), and then introduce the fix.
2. This probably belongs to another PR.

Reply:
Yep, it's fixing a bug.
I fixed the test. It is necessary to create a new PR only for this fix?

Reply:
It's a matter of good practices. PRs should have minimal content so they can be reviewed easily and merged quickly.

If you piggyback on the new feature to introduce bug fixes, you make it harder to review the changes and you make it harder to revert the changes if that is needed. The general rule of thumb is that if you can separate them, then you should separate them.

Have a small PR to fix the bug and introduce the test case. This should be merged quickly, and it will make it easier to review the new feature.

Reply:
Okay I got this. But in this case the other files in this PR (html file) dependents on this function so I wont separate this content.
I will take care about this at future PRs.

Reply:
alright

Reply:
I'm not sure now what you meant when you say you'd take care of it at a future PR? If you need to include it here, then you have to introduce a test that recreates the issue.